setup.py

```diff
@@ -29,14 +29,19 @@
 
 from setuptools.command.build_ext import build_ext as BuildExtension
 
+# Same list as build_system.requires in pyproject.toml
+install_packages(["setuptools >= 61.0", "cmake>=3.21", "pybind11", "ninja"])
+
 os.environ["NVTE_PROJECT_BUILDING"] = "1"
 
 if "pytorch" in frameworks:
+    install_packages(["torch>=1.13"])
     from torch.utils.cpp_extension import BuildExtension
```

Comment:
Does this work with build isolation? Even if we reinstall PyTorch with Pip, I'm not sure if the virtual environment has access to the system packages.